sonar-server/Dockerfile

```diff
@@ -1,18 +1,15 @@
-from dockerfile/java
-maintainer Tiago Pires, tandrepires@gmail.com
+FROM dockerfile/java
```

Comment:
since you're updating, and `java` is deprecated, can you pls update to `openjdk`? thanks